custom/icds/utils/custom_sms_report.py

```diff
@@ -0,0 +1,43 @@
+import json
+from dimagi.utils.couch.cache.cache_core import get_redis_client
+
+
+class CustomSMSReportTracker(object):
+    def __init__(self):
+        self.client = get_redis_client()
+
+    @property
+    def report_key(self):
+        return 'custom-sms-report-tracker'
```

Comment:
this should also have domain in it, for if its used by different domains (unlikely but possible for ex when testing in a domain apart from icds-cas).